Lib/fontTools/fontBuilder.py

```diff
@@ -838,7 +854,10 @@ def addFvar(font, axes, instances):
         axis = Axis()
         axis.axisTag = Tag(tag)
         axis.minValue, axis.defaultValue, axis.maxValue = minValue, defaultValue, maxValue
-        axis.axisNameID = nameTable.addName(tounicode(name))
+        if isinstance(name, basestring):
```

Comment:
Probably best to use `str`.

Ideally, the py23 import should be eliminated completely (ugh, there are two of them...). I'm fairly sure the `Tag` cast is as superfluous as `tounicode`.